### DISCUSSION TITLE
Add iconsax

### DISCUSSION THREAD

**Discussion by raibann** [ng-icons/ng-icons#74] [Ideas]
Can you add iconsax for more?

**Comment by ashley-hunter** [comment]
Hi, thanks for the suggestion. That iconset does look nice and I would be happy to add it, however, unfortunately, it seems very vague about what license the iconset is provided under. There is no license information in the repository https://github.com/lusaxweb/iconsax and the website mentions licensing but is not specific about the actual license. If we can get some clarification in this area I would happily add it, however the repositories seem to have been inactive since 2021.
